src/game/client/components/nameplates.cpp

```diff
@@ -109,6 +111,37 @@ void CNamePlates::RenderNameplate(
 			}
 		}
 
+		if(g_Config.m_ClAfkMark && m_pClient->m_aClients[ClientID].m_Afk)
+		{
+			const char *afkMark = "!";
```

Comment:
```suggestion
			const char *AfkMark = "!";
```
We don't do camelCase